iXBRLViewerPlugin/viewer/src/js/tableExport.js

```diff
@@ -16,11 +16,23 @@ export class TableExport {
         $('table', iframe).each(function () {
             const table = $(this);
             if (table.find(".ixbrl-element").length > 0) {
-                table.css("position", "relative");
-                const exporter = new TableExport(table, reportSet);
-                $('<div class="ixbrl-table-handle"><span>Export table</span></div>')
-                    .appendTo(table)
-                    .click(() => exporter.exportTable());
+                const containsAbsolute = Array.from(this.querySelectorAll("*"))
+                    .some((elt) => getComputedStyle(elt).getPropertyValue('position') === 'absolute');
+
+                // Don't add handles if the table contains absolutely
+                // positioned elements, as doing so will probably interfere
+                // with layout.
+                //
+                // Where a table has absolutely positioned content, it's quite
+                // likely that the table tag itself is positioned in a random
+                // place.
```

Comment:
I worry this will break/remove the handle from Workiva-generated documents, which use `position:absolute` on all `<div>` within a `<table>` but I believe position the `<table>` in a specific/non-random place ([test](https://github.com/Arelle/ixbrl-viewer/files/14559510/p-2024-03-11-en.zip)).

Reply:
I saw a link to [this blog post](https://frontendmasters.com/blog/what-you-need-to-know-about-modern-css-spring-2024-edition/#anchor-positioning) over the weekend, which seemed promising:
> Once you can use this freely, you’ll have to care less about exact DOM positioning of elements (aside from accessibility concerns). The way it is now, the element you want to position relative to another has to be a child element and for there to be a positioning context to work within. This can dictate where elements go in the DOM, whether or not that makes sense.

Looks like Firefox [intends to prototype](https://bugzilla.mozilla.org/show_bug.cgi?id=1838746), which is promising.